packages/loader/driver-utils/src/index.ts

```diff
@@ -22,6 +22,7 @@ export {
 	getRetryDelaySecondsFromError,
 	isOnline,
 	LocationRedirectionError,
+	RequestRedirectionError,
```

Comment:
Why this new error type. We already have the error type for this. "LocationRedirectionError". We should use that only and not create duplicate error types.

Reply:
I had to make a new error type for this redirect as the previously existing LocationRedirectionError was asking for things from IResolvedUrl which I did not have access to. I could just "fake" the data from IResolvedUrl that I don't have access to if you think that  is better than having a new error

Reply:
Found `OdspRedirectError` on `packages/utils/odsp-doclib-utils/src/odspErrorUtils.ts`, so using that instead!